src/bzflag/BackgroundRenderer.cxx

```diff
@@ -851,18 +852,18 @@ void BackgroundRenderer::drawSkybox()
         glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_T, GL_CLAMP_TO_EDGE);
         glBegin(GL_TRIANGLE_STRIP);
         {
-            glTexCoord2fv(txcds[0]);
+            glTexCoord2fv(glm::value_ptr(txcds[0]));
```

Comment:
Any particular reason to use glm vectors for the vertex and texture coordinates, but not for the color component, whereas you do use a glm vector for color components in other files?

Reply:
No reason. Only they did not help me later. I think we should always use glm vectors but need some help 'cause is a lot of work and test